src/backend/revoker/indexer/queryAttestation.test.ts

```diff
@@ -0,0 +1,314 @@
+
+        for await (const match of aFewAttestations) {
+          expect(match).toBeDefined();
+        }
+
+        expect(got.post).toHaveBeenCalledTimes(1);
+      });
+
+      it('should continue requesting from the Indexer until querying all matches', async () => {
+        const count = Math.floor(QUERY_SIZE * 3.33);
+        postResponse = {
+          data: {
+            blocks: {
+              totalCount: count,
+              nodes: mockAttestations(QUERY_SIZE).map((b) => ({ ...b })),
+            },
+          },
+        };
+        const aLotOfAttestations = queryExpiredAttestations();
+
+        for await (const match of aLotOfAttestations) {
+          expect(match).toBeDefined();
+        }
+
+        expect(got.post).toHaveBeenCalledTimes(5);
+      }, 10000);
+      it('should request from the Indexer using the expected queries', async () => {
+        const count = Math.floor(QUERY_SIZE * 3.33);
+        postResponse = {
+          data: {
+            blocks: {
+              totalCount: count,
+              nodes: mockAttestations(QUERY_SIZE).map((b) => ({ ...b })),
+            },
+          },
+        };
+        const aLotOfAttestations = queryExpiredAttestations();
+
+        for await (const match of aLotOfAttestations) {
+          expect(match).toBeDefined();
+        }
+
+        expect(got.post).toHaveBeenCalledTimes(5);
+
+        // `fromDate` is only 0 on the app's start
+        // const timeZero = new Date(0);
+        const aYearAgo = new Date(frozenNow.valueOf());
+        aYearAgo.setFullYear(aYearAgo.getFullYear() - 1);
+
+        const buildAttestationQuery = buildAttestationQueries(
+          fromDate,
+          aYearAgo,
+        );
+
+        const { calls: postRequests } = jest.mocked(got.post).mock;
+
+        // @ts-expect-error because TS infers wrong parameters
+        expect(postRequests[0][1]).toMatchObject({
+          json: { query: buildAttestationQuery(0) },
+        });
+        // @ts-expect-error because TS infers wrong parameters
+        expect(postRequests[1][1]).toMatchObject({
+          json: { query: buildAttestationQuery(0) },
+        });
+        // @ts-expect-error because TS infers wrong parameters
+        expect(postRequests[2][1]).toMatchObject({
+          json: { query: buildAttestationQuery(QUERY_SIZE) },
+        });
+        // @ts-expect-error because TS infers wrong parameters
+        expect(postRequests[3][1]).toMatchObject({
+          json: { query: buildAttestationQuery(QUERY_SIZE * 2) },
+        });
+        // @ts-expect-error because TS infers wrong parameters
+        expect(postRequests[4][1]).toMatchObject({
+          json: { query: buildAttestationQuery(QUERY_SIZE * 3) },
+        });
+      }, 10000);
+      it("should yield attestations in the 'AttestationInfo' interface", async () => {
+        const cousinNumbers = [1, 2, 3, 5, 7, 13];
+
+        const mockedMatches: QueriedAttestation[] = cousinNumbers.map((n) => {
+          return {
```

Comment:
```suggestion
        const mockedMatches: QueriedAttestation[] = cousinNumbers.map((n) => ({
```
Are there benefits to not using a shorthand form here and below?

Reply:
Used shorthand now.